fmn/delivery/backends/debug.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import print_function
```

Comment:
This could use a copyright too - I won't mention more if there are more, please ensure the new files have copyright headers.